core/src/main/java/hudson/model/Node.java

```diff
@@ -115,6 +118,18 @@ public boolean isHoldOffLaunchUntilSave() {
     }
 
     /**
+     * {@inheritDoc}
+     * @since 1.635
+     */
+    @Override
+    public void save() throws IOException {
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins != null) {
+            jenkins.updateNode(this);
+        }
```

Comment:
The default implementation of `Saveable.save()` for most cases will be a no-op if there is no place to save the configuration. Without a `Jenkins` there is no JENKINS_HOME and thus there is no config file to save, so the "normal" thing is to just do a no-op